Panic if SSH key not found

**Describe the bug**
`lagoon login` tries to open `/home/scott/.ssh/id_rsa` and panics if that file is not found.

**To Reproduce**
Steps to reproduce the behavior:
1. If you have a file in `~/.ssh/id_rsa`, move it elsewhere temporarily.
2. run `lagoon login`

**Expected behavior**
SSH supports multiple key formats. According to `man ssh`:
```
The default is ~/.ssh/id_dsa, ~/.ssh/id_ecdsa, ~/.ssh/id_ed25519 and ~/.ssh/id_rsa.
```
I'm using ed25519.

So I guess it should try to load the keys in the same order the SSH client does..?

**Screenshots**
```
$ lagoon version
Version: 0.6.1
Build: 2019-12-06T04:46:13+0000
$ lagoon login
panic: open /home/scott/.ssh/id_rsa: no such file or directory

goroutine 1 [running]:
github.com/amazeeio/lagoon-cli/cmd.publicKey(0xc0000b1480, 0x17, 0xc00012db00, 0x1, 0x1, 0xc0000b1480)
	/go/src/github.com/amazeeio/lagoon-cli/cmd/login.go:31 +0x751
github.com/amazeeio/lagoon-cli/cmd.loginToken(0x0, 0x0)
	/go/src/github.com/amazeeio/lagoon-cli/cmd/login.go:76 +0xdc
github.com/amazeeio/lagoon-cli/cmd.validateToken(0xc0000af2e8, 0x8)
	/go/src/github.com/amazeeio/lagoon-cli/cmd/root.go:272 +0x4f
github.com/amazeeio/lagoon-cli/cmd.glob..func28(0xe59340, 0xe86508, 0x0, 0x0)
	/go/src/github.com/amazeeio/lagoon-cli/cmd/login.go:21 +0x5f
github.com/spf13/cobra.(*Command).execute(0xe59340, 0xe86508, 0x0, 0x0, 0xe59340, 0xe86508)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xe5bb40, 0x0, 0x0, 0x8cc213)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/amazeeio/lagoon-cli/cmd.Execute()
	/go/src/github.com/amazeeio/lagoon-cli/cmd/root.go:58 +0x42
main.main()
	/go/src/github.com/amazeeio/lagoon-cli/main.go:8 +0x20
```

**Desktop (please complete the following information):**
 - OS: Debian Testing (Linux)

**Additional context**
I'm using the linux-amd64 binary downloaded from the github releases page.

--- a/cmd/login.go
+++ b/cmd/login.go
@@ -26,11 +26,6 @@ var loginCmd = &cobra.Command{
 func publicKey(path string, skipAgent bool) (ssh.AuthMethod, func() error) {
 	noopCloseFunc := func() error { return nil }
 
-	key, err := ioutil.ReadFile(path)
-	if err != nil {
-		panic(err)
-	}
-
 	if skipAgent != true {
 		// Connect to SSH agent to ask for unencrypted private keys
 		if sshAgentConn, err := net.Dial("unix", os.Getenv("SSH_AUTH_SOCK")); err == nil {
@@ -43,6 +38,11 @@ func publicKey(path string, skipAgent bool) (ssh.AuthMethod, func() error) {
 				return ssh.PublicKeysCallback(sshAgent.Signers), sshAgentConn.Close
 			}
 		}
+	}
+
+	key, err := ioutil.ReadFile(path)
+	if err != nil {
+		panic(err)
 	}
 
 	// Try to look for an unencrypted private key

